Sources/Security.php

```diff
@@ -906,6 +906,11 @@ function allowedTo($permission, $boards = null, $any = false)
 	if (empty($user_info))
```

Comment:
Why not just add ` || !isset($user_info['permissions'])` to this line instead?

Reply:
Then I'd lose the comment.

Reply:
😆 Well, I suppose you could move the comment up under the existing one.